cmd/lncli/cmd_open_channel.go

```diff
@@ -817,6 +818,13 @@ var batchOpenChannelCommand = cli.Command{
 				"transaction when storing it to the local " +
 				"wallet after publishing it",
 		},
+		cli.StringFlag{
+			Name:  "coin_selection_strategy",
+			Usage: "(optional) The strategy to use for selecting " +
```

Comment:
This should mention that when using the parameter, the globally (in `lnd.conf`) configured strategy is overwritten. I think that's one more reason to use an enum on the RPC side (see my other comments).